initrd/bin/network-init-recovery

```diff
@@ -8,13 +8,30 @@ TRACE "Under /bin/network-init-recovery"
 ifconfig lo 127.0.0.1
 
 network_modules="e1000 e1000e igb sfc mdio mlx4_core mlx4_en"
-for module in `echo $network_modules`; do
+for module in $(echo $network_modules); do
 	if [ -f /lib/modules/$module.ko ]; then
 		insmod /lib/modules/$module.ko
 	fi
 done
 
-if [ -e /sys/class/net/eth0 ]; then
+#Tethering over Android 11+ PoC
+if [ -e /lib/modules/rndis_host.ko ]; then
+	#first enable USB controllers
+	enable_usb
+	network_modules="mii usbnet cdc_ecm cdc_subset cdc_eem cdc_ether cdc_ncm rndis_host"
+
+	for module in $(echo $network_modules); do
+		if [ -f /lib/modules/$module.ko ]; then
+			insmod /lib/modules/$module.ko
+		fi
+	done
+	#other prep needed
```

Comment:
Can we delete this `#other prep needed` comment or is this a placeholder for something?